vision/cloud-client/detect/detect.py

```diff
@@ -661,6 +661,95 @@ def detect_document_uri(uri):
 # [END vision_fulltext_detection_gcs]
 
 
+# [START vision_fulltext_detection_pdf]
+def detect_pdf_document(path):
+    """Detects document features in a PDF/TIFF/GIF file."""
+    from google.cloud import vision_v1p4beta1 as vision
+    client = vision.ImageAnnotatorClient()
+
+    with io.open(path, 'rb') as pdf_file:
+        content = pdf_file.read()
+
+    # Can be other like 'image/tiff' or 'image/gif'
+    mime_type = 'application/pdf'
+    file_bytes_input_config = vision.types.InputConfig(
```

Comment:
Let's just name it `input_config` to match the method's parameter.

Reply:
Done